engines/mtropolis/detection_tables.h

```diff
@@ -108,9 +108,127 @@ static const MTropolisGameDescription gameDescriptions[] = {
 		0,
 	},
 
+	{ // Obsidian Macintosh demo
+		{
+			"obsidian",
+			"Demo",
+			{
+				{ "Obsidian Demo", 0, "abd1b5e7ac133f4c4b8c45ac67a4c44d", 920832 },
+				{ "Basic.rPP", 0, "cb567ec1423a35903d8d5f458409e681", 210432 },
+				{ "Experimental.rPP", 0, "26aa5fe1a6a152ade74e23a706673c50", 102016 },
+				{ "Extras.rPP", 0, "c0e4c0401f2107ba3a3b7d282a76d99b", 377600 },
+				{ "mCursors.cPP", 0, "a52e2aaf3b1a5c7d93a2949693bca694", 13312 },
+				{ "mNet.rPP", 0, "ed5d998e7db6daae1f24bb124cc269aa", 134784 },
+				{ "Obsidian.cPP", 0, "6da7babae9725a716f27f9f4ea382e92", 7552 },
+				{ "RSGKit.rPP", 0, "c359e3c932b09280d1ccf21f8fb52bd7", 668160 },
+				{ "Obs Demo Large w Sega", 0, "34efe95fc32dcf0a2a90bf6854c08e95", 98953679 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformMacintosh,
+			ADGF_DEMO,
+			GUIO0()
+		},
+		GID_OBSIDIAN,
+		0,
+		0,
+	},
+
+	{ // Obsidian PC demo from PC Gamer Disc 2.12 (1997-01)
+		{
+			"obsidian",
+			"Demo",
+			{
+				{ "OBSIDIAN.EXE", 0, "b6fb0e0df88c1524bcd0c5de9f5e882c", 750080 },
+				{ "OBSIDIAN.R95", 0, "5361ef93e36d722665594b724e0018fd", 183296 },
+				{ "TEXTWORK.R95", 0, "96346d39c4bb04f525edbf06ffe047e0", 148992 },
+				{ "EXPRMNTL.R95", 0, "aa0431c2be37e33883747c61d3e980ff", 108544 },
+				{ "MCURSORS.C95", 0, "47cf6abb95f3c43cdcbdf7ea1de3478d", 145920 },
+				{ "OBSIDI~1.MPL", 0, "643a989213b42cbac319d04676447624", 29096880 },
+				AD_LISTEND
+			},
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_DEMO | ADGF_UNSTABLE,
+			GUIO0()
+		},
+		GID_OBSIDIAN,
+		0,
+		0,
+	},
+
+	{ // Obsidian PC demo
```

Comment:
In Discord, you had more info on these demos like the dates. Might be helpful to include them.